src/disco/components/Addon.js

```diff
@@ -41,6 +46,15 @@ export default class Addon extends React.Component {
     return JSON.stringify({id, name, headerURL, footerURL, textcolor, accentcolor});
   }
 
+  getError() {
+    const { status } = this.props;
+    const errorMessage = this.props.errorMessage || _('An unexpected error occurred');
+    return status === ERROR ? (<div className="error">
```

Comment:
I think we should be namespacing these classes. Maybe we should be prefixing the component name?

`Addon-error`, `Addon-error-message`, `Addon-error-close`?